### PR TITLE
Bump version to 0.4.9

### PR DESCRIPTION
**Pull request by zanieb**

### CODE REVIEW

**Comment by zanieb** [comment]
Reverting https://github.com/astral-sh/uv/pull/5927 because it changed a bunch of versions and CI is failing.

**Review by charliermarsh — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1524,7 +1524,7 @@ jobs:
           cargo run --bin uv -- pip compile scripts/requirements/airflow.in --universal --exclude-newer 2024-08-08 --cache-dir .cache
 
       - name: "Build benchmarks"
-        run: cargo codspeed build --profile profiling --features codspeed -p bench
```
Comment (charliermarsh) [inline .github/workflows/ci.yml:1527]: Was this intended to be removed?

Comment (zanieb) [reply .github/workflows/ci.yml]: Yeah, but it's restored now that https://github.com/astral-sh/uv/pull/7276 is merged

**Comment by zanieb** [comment]
https://github.com/astral-sh/uv/pull/7276 reverted the lock changes so I've rebased onto `main` and dropped the revert of https://github.com/astral-sh/uv/pull/5927

**Pull request merged**